rbd/rbd_sparsify.go

```diff
@@ -0,0 +1,82 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package rbd
+
+/*
+#cgo LDFLAGS: -lrbd
+#include <errno.h>
+#include <stdlib.h>
+#include <rbd/librbd.h>
+#include <rados/librados.h>
```

Comment:
Double check: what was librados.h needed for?

Reply:
Not required, removed it.